networkit/cpp/graph/test/GraphToolsGTest.cpp

```diff
@@ -1039,4 +1033,12 @@ TEST_P(GraphToolsGTest, testEdgesSortedByWeight) {
         EXPECT_TRUE(G.checkConsistency());
     }
 }
+
+TEST_F(GraphToolsGTest, testEdgesRandomizer) {
+    Aux::Random::setSeed(1, true);
+    DorogovtsevMendesGenerator generator(1000);
+    Graph G1 = generator.generate();
+    Graph G(G1, true, false);
+    GraphTools::randomizeWeights(G);
+}
```

Comment:
Once the deterministic version works in parallel, we should also provide meaningful tests. Here we are only checking that the function runs, not that it actually works.